src/main/kotlin/com/hypto/iam/server/service/PasscodeService.kt

```diff
@@ -151,7 +154,15 @@ class PasscodeServiceImpl : KoinComponent, PasscodeService {
             .templateData(gson.toJson(templateData))
             .destination(Destination.builder().toAddresses(email).build())
             .build()
-        sesClient.sendTemplatedEmail(emailRequest)
+        try {
+            sesClient.sendTemplatedEmail(emailRequest)
+        } catch (e: SesException) {
+            if (e.statusCode() == HttpStatusCode.BAD_REQUEST) {
+                throw BadRequestException("Bad request values")
```

Comment:
Can we change the message to something else, because a user might be confused from this message.